File: hw0603uitask2/src/main/java/com/tashad16a/gmail/hw0603ui/uigooglepodcasts/fragment/PageFragment.java
package com.tashad16a.gmail.hw0603ui.uigooglepodcasts.fragment;

import android.os.Bundle;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import com.tashad16a.gmail.hw0603_ui.R;

public class PageFragment extends Fragment {

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
    }

    @Override
    public View onCreateView(LayoutInflater inflater,
                             ViewGroup container,
                             Bundle savedInstanceState) {
        return inflater.inflate(R.layout.activity_pg_for_you_fragment, container, false);
    }
}

File: hw1103compaundviewnavigationdrawer/src/full/java/com/tashad16a/gmail/full/fragment/PageFragment.kt
package com.tashad16a.gmail.full.fragment

import android.os.Bundle
import android.support.annotation.NonNull
import android.support.annotation.Nullable
import android.support.v4.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import com.tashad16a.gmail.hw1103compaundviewnavigationdrawer.R

class PageFragment : Fragment() {

    @Nullable
    override fun onCreateView(
        @NonNull inflater: LayoutInflater,
        container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        return inflater.inflate(R.layout.fragment_page, container, false)
    }
}